README.md

```diff
@@ -29,6 +29,26 @@ If you are interested in using AdvancedHMC.jl through a probabilistic programmin
 
 ## A minimal example - sampling from a multivariate Gaussian using NUTS
 
+In this section we demonstrate a minimal example of sampling from a multivariate Gaussian (10 dimensional) using the no U-turn sampler (NUTS). Below we describe the major components of the Hamiltonian system which are essential to sample using this approach:
+
+- **Metric**: In many sampling problems the sample space is usually associated with a metric, that allows us to measure the distance between any two points, and other similar quantities. In the example in this section, we use a special metric called the **Euclidean Metric**, represented with a `D × D` matrix from which we can compute distances. 
+<details>
+ <summary>Further details about the Metric component</summary>
+ The Euclidean metric is also known as the mass matrix in the physical perspective. For available metrics refer <a href="https://github.com/SaranjeetKaur/AdvancedHMC.jl#hamiltonian-mass-matrix-metric">Hamiltonian mass matrix</a>.
```

Comment:
We need to avoid these hard-coded links: `https://github.com/SaranjeetKaur/AdvancedHMC.jl`. There are a few more in other places.